Makefile

```diff
@@ -114,7 +120,9 @@ $(WIN64).zip: compile-all dw-all
 	# Set new root folder
 	sed -i 's/^\$$script\:OH_DIR\=\".\"/\$$script\:OH_DIR\=\"oh\"/g' $(WIN64)/oh.ps1
 	sed -i 's/set\ OH_DIR=\".\"/\set\ OH_DIR\=\"oh\"/g' $(WIN64)/oh.bat
+	# copy manuals
 	cp *.pdf $(WIN64)/doc
+	# create archive
```

Comment:
is there a reason why here is "create archive" and others have "create package" ?

Reply:
Right, fixed !